lib/beaker-hostgenerator/data.rb

```diff
@@ -859,6 +859,24 @@ def osinfo
             'template' => 'win-2012r2-x86_64'
           }
         },
+        'windows2012r2_ja-64' => {
```

Comment:
@waynr do you have any thoughts on the pattern proposed here for language specific VMs?  Glenn has run with `_ja` - but want to make sure that jives with any plans you had for the future.


Any input from you @joshcooper ?